ketcher-autotests/playwright.config.ts

```diff
@@ -0,0 +1,126 @@
+import * as dotenv from 'dotenv';
+import type { PlaywrightTestConfig } from '@playwright/test';
+import { devices } from '@playwright/test';
+import {
+  REMOTE_URL,
+  STANDALONE_URL,
+  DEFAULT_KETCHER_STANDALONE_URL,
+  MODES,
+} from './constants';
+
+dotenv.config();
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+
+function baseURL(): string {
+  if (!process.env.MODE || !process.env.KETCHER_URL) {
+    return DEFAULT_KETCHER_STANDALONE_URL;
+  }
+
+  if (process.env.MODE === MODES.STANDALONE) {
+    return `${process.env.KETCHER_URL}${STANDALONE_URL}`;
+  }
+
+  return `${process.env.KETCHER_URL}${REMOTE_URL}`;
+}
+
+const config: PlaywrightTestConfig = {
+  testDir: './tests',
+  /* Maximum time one test can run for. */
+  timeout: 100 * 1000,
+  expect: {
+    /**
+     * Maximum time expect() should wait for the condition to be met.
+     * For example in `await expect(locator).toHaveText();`
+     */
+
+    // toHaveScreenshot: {
+    //   /* An acceptable ratio of pixels that are different to the total amount of pixels, between 0 and 1. */
+    //   maxDiffPixelRatio: 0.01,
+    // },
+    timeout: 10000,
+  },
+  /* Run tests in files in parallel */
+  fullyParallel: true,
+  /* Fail the build on CI if you accidentally left test.only in the source code. */
+  forbidOnly: !!process.env.CI,
+  /* Retry on CI only */
+  retries: process.env.CI ? 2 : 0,
+  /* Opt out of parallel tests on CI. */
+  workers: process.env.CI ? 1 : undefined,
+  /* Reporter to use. See https://playwright.dev/docs/test-reporters */
+  reporter: 'html',
+  /* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
+  use: {
+    /* Maximum time each action such as `click()` can take. Defaults to 0 (no limit). */
+    actionTimeout: 0,
+    viewport: { width: 1920, height: 1080 },
+    /* Base URL to use in actions like `await page.goto('/')`. */
+    baseURL: baseURL(),
+
+    /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
+    trace: 'on-first-retry',
+  },
+
+  /* Configure projects for major browsers */
+  projects: [
+    {
+      name: 'chromium',
```

Comment:
Lets leave only chromium to speed things up, since QAs only use it for checking

Reply:
fixed